src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1985,8 +1998,13 @@ export default class GeFormRenderer extends LightningElement{
             this.CUSTOM_LABELS.geTextUpdating :
             this.CUSTOM_LABELS.geTextSaving;
 
-        const upsertResponse = await upsertDataImport({ dataImport: dataImportFromFormState });
-        this.updateFormState(upsertResponse);
+            await upsertDataImport({
```

Comment:
I think since the async keyword is being used for this function the call to upsertDataImport should be wrapped in a try catch block instead of using the .then and .catch here.